client.c

```diff
@@ -53,9 +53,7 @@ int n_clients = 0;
 #else
 #define NO_INVALID_ADDRESSES 1
 #endif
-struct sockaddr invalidAddresses[NO_INVALID_ADDRESSES] = { 0 };
-/* Is this way of initializing to 0 compatibel with all compilers in use?
- * Otherwise do a memset() in clientmgr() */
+struct sockaddr invalidAddresses[NO_INVALID_ADDRESSES];
```

Comment:
Indeed...

On Tue, Jan 19, 2016 at 4:48 PM, Pavel <notifications@github.com> wrote:

> In client.c <https://github.com/pgul/binkd/pull/5#discussion_r50128822>:
>
> > @@ -53,9 +53,7 @@ int n_clients = 0;
> >  #else
> >  #define NO_INVALID_ADDRESSES 1
> >  #endif
> > -struct sockaddr invalidAddresses[NO_INVALID_ADDRESSES] = { 0 };
> > -/* Is this way of initializing to 0 compatibel with all compilers in use?
> > - * Otherwise do a memset() in clientmgr() */
> > +struct sockaddr invalidAddresses[NO_INVALID_ADDRESSES];
>
> static?
> We don't need to export this variable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pgul/binkd/pull/5/files#r50128822>.
>



-- 
Met vriendelijke groet / Best regards,
Wilfred van Velzen